mission.html

```diff
@@ -4,88 +4,102 @@
-            <p>The technology to do better is here today.</p>
-            <p><strong>Together, we aim to bring an end to the makeshift.</strong></p>
-        </div>
-    </section>
-    <section class="page-row" id="people">
-        <picture id="people-img">
+    <page-header></page-header>
+    <main>
+        <section class="page-row">
+            <div class="title">
+                <h1>Search and rescue <br /> has been left behind.</h1>
+            </div>
+            <div class="subtitle">
+                <p><i>We're changing that.</i></p>
+            </div>
+        </section>
+        <section class="page-row" id="dev">
+            <div>
+                <h1>ResQ Systems is developing an affordable end-to-end Unmanned Aerial System to revolutionise SAR and
+                    other industries.</h1>
+            </div>
+            <picture>
+                <img src="/img/props.png" />
+            </picture>
+            <div>
+                <p>Stirred to action by the generational leaps UAS technology has taken, our UK-based team gathered with
+                    one goal in mind: to put an end to imperfect solutions for mountain and lowland rescuers.</p>
+                <p>Drones are the biggest multiplier available to rescue teams searching for those in trouble - but they
+                    are forced to make do with mass-produced, commercial systems that leave little room for adaptation,
+                    little time for flying and <strong>little budget left afterwards.</strong></p>
+                <p>Drone operators are forced to adhere to the limitations of their systems, wasting valuable time and
+                    another pair of eyes, another pair of hands.</p>
+                <p>The technology to do better is here today.</p>
+                <p><strong>Together, we aim to bring an end to the makeshift.</strong></p>
+            </div>
+        </section>
+        <section class="page-row" id="people">
+            <picture id="people-img">
 
-        </picture>
-        <div>
-            <h1>4785+</h1>
-            <h3>Mountain and Lowland Rescue callouts in 2024* alone</h3>
-            <p>The countryside is an escape for many. Social media and word-of-mouth inspires thousands to explore our beautiful natural landscapes, especially the more dangerous spots. More people brings more emergencies, and rescue teams across the country must respond to each and every one. Sometimes, over 75% of a team's callouts are for injured casualties**. When casualties are alone, trapped, cold, in severe danger - <strong>every second counts.</strong></p>
-            <p><small>* 4785 individual team callouts between Mountain Rescue England & Wales and Mountain Rescue Scotland between 1 Jan - 31 Dec 2024. Detailed statistics from other groups for 2024 yet to release.<br/>
```

Comment:
generally I would avoid <br> tags, this is realistically a list, as such should be marked up as a list, but this is a very minor markup semantic.